.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and test
+
+on:
+  push:
+
+jobs:
```

Comment:
Instead of having two jobs, you can create a [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) which reduces duplication and allows running the two tests concurrently.

Reply:
The two jobs are run in parallel too. I didn't use a matrix because I thought that we'll either reunify both versions or the CI jobs will divert.

Reply:
Yeah, I noticed that, I don't have much experience with GitHub Actions.